pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -1,5 +1,5 @@
 # GENERATED by ./pkgs/applications/editors/vim/plugins/update.py. Do not edit!
-{ lib, buildVimPlugin, buildNeovimPlugin, fetchFromGitHub, fetchgit }:
+{ lib, buildVimPlugin, buildNeovimPlugin, fetchFromGitHub }:
```

Comment:
Rebased and the only change is this, to cause an eval failure if someone tries to add non-github plugin

Reply:
Will the updater just readd it though?

Reply:
If so, just something we have to keep an eye out for on updates until it gets changed.

Reply:
Also, accidentally gave you a sneak-pick for the new generated.json

https://github.com/NixOS/nixpkgs/blob/90df3069476a28e3866ef8163c36d83184a88367/pkgs/applications/editors/vim/plugins/generated.json